sqlglot/dialects/tsql.py

```diff
@@ -4,20 +4,65 @@
 from sqlglot.generator import Generator
 from sqlglot.helper import list_get
 from sqlglot.parser import Parser
+from sqlglot.time import format_time
 from sqlglot.tokens import Tokenizer, TokenType
 
 
+def tsql_format_time_lambda(exp_class, mapping=None, default=None):
+    corrected_mapping = {
```

Comment:
make this a constant?

Reply:
Constant inside the method or constant in the class and passed along to the method?

Reply:
make it a constant at the tsql file level, and just reference it

Reply:
put it at file level and gave it a more generic name. function has a switch to use the overwrite for full expressions.